components/vc/rear/src/mcManager.c

```diff
@@ -120,6 +133,17 @@ static void mcManager_periodic_100Hz(void)
     float32_t torque_command = 0.0f;
     mcManager_enable_E enable = MCMANAGER_DISABLE;
     CAN_prechargeContactorState_E contactor_state = CAN_PRECHARGECONTACTORSTATE_SNA;
+    float32_t motor_rpm = 0.0f;
+    const bool speed_valid = CANRX_get_signal(ASS, PM100DX_motorSpeedCritical, &motor_rpm) == CANRX_MESSAGE_VALID;
```

Comment:
you should add a macro like `CANRX_get_signal` that does this comparison for you, like `CANRX_get_signal_validity(BUS, SIGNAL, DEST)` that returns a bool if valid.

Also you should be able to use macro-magic to make it so you don't have to specify the bus